client/client.go

```diff
@@ -58,7 +58,7 @@ import (
 )
 
 // DefaultVersion is the version of the current stable API
-const DefaultVersion string = "1.23"
+const DefaultVersion string = "1.25"
```

Comment:
We should probably put a comment here, and in https://github.com/docker/docker/blob/70459d6f1ec98fa0aa0bf0d873838a9c4b5d3236/api/common.go#L24, that both should be kept in sync

Also, wondering why bumping these versions isn't described in https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md#22-update-api-version-on-master